src/frontend/src/api/ResourceCollection.ts

```diff
@@ -30,6 +30,10 @@ export default class ResourceCollection {
         return combineUrls(this.contract(projectId, contractId), 'resources', 'personnel-collection');
     }
 
+    personnelUpdate(projectId: string, contractId: string, personnelId: string): string {
```

Comment:
Resource collection should not care about the "action" you're performing on a resource. This method could be merged with `personnel` with optional `personnelId` parameter. E.g. `personnel(projectId: string, contractId: string, personnelId?: string) { return combineUrls(this.contract(projectId, contractId), 'resources', 'personnel', personnelId);`